src/main/kotlin/it/polpetta/api/jenkins/Version.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * jkk
+ * Copyright (C) 2019 - 2020 Davide Polonio <poloniodavide@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package it.polpetta.api.jenkins
+
+/**
+ * Class that wrap up the Jenkins version.
+ */
+interface Version {
+
+    /**
+     * @return whenever this particular Jenkins version is valid
```

Comment:
That's a tad generic as a definition, care to provide an example of a valid/invalid version?

Reply:
If the server replies with something like (-1) like the previous library this would tell you that something is not going really well :grin:

Reply:
I'd add this example to the documentation imho :)

Reply:
On top of that, it could be used in the future to check if the version of the Jenkins server is supported!

Reply:
The I'd change this documentation to `@return whether this particular Jenkins version is valid for the purposes of the implementation`

Reply:
Done :+1: